libs/blocks/tabs/tabs.js

```diff
@@ -57,7 +57,28 @@ export function getRedirectionUrl(linkedTabsList, targetId) {
   return currentUrl;
 }
 
-function changeTabs(e) {
+const generateStorageName = (tabId) => {
+  const { pathname } = window.location;
+  return `${pathname}/${tabId}-tab-state`;
+};
+
+const loadActiveTab = (config) => {
+  if (config['remember-selected-tab'] !== 'on') return 0;
+
+  const tabId = config['tab-id'];
+  return sessionStorage.getItem(generateStorageName(tabId));
```

Comment:
fallback to 0 if it doesn't exist in sessionStorage?

Reply:
This is the method call.
```
const activeTabIndex = loadActiveTab(config);
if (activeTabIndex) config['active-tab'] = activeTabIndex;
```
Zero is not the valid tab index, it starts from 1.
Return value 0 just means that the value configured in the document for "active-tab" will be used and sessionStorage will be ignored.